### DISCUSSION TITLE
I am a University Student need help

### DISCUSSION THREAD

**Discussion by extremeaccess** [community/community#67022] [GitHub Education]
### Select Topic Area

Question

### Body

Subject: Request for GitHub Student Pack

Dear GitHub Support Team,

I hope this message finds you well. I am writing as a dedicated university student and a passionate enthusiast in the field of software development. I have been an active GitHub user for quite some time now, and I have heard about the incredible benefits offered through the GitHub Student Pack.

However, I regret to inform you that I have not yet received access to the GitHub Student Pack, despite meeting the eligibility criteria as a verified student. As a student, access to this pack would greatly assist me in my academic pursuits and personal projects. The tools and resources included in the pack would not only enhance my learning experience but also enable me to contribute effectively to open-source projects and collaborate with peers.

I kindly request your assistance in expediting the verification process and providing me with access to the GitHub Student Pack. Your support would greatly empower my educational journey and help me contribute to the developer community more effectively.

Thank you for your attention to this matter, and I look forward to your prompt response.

Sincerely,

 Name: Aymen
 University Email Address: mc210201843
GitHub Username: extremeexcess

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @extremeaccess , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you to the GitHub Education category. Good luck!
